tests/suit/framework/openstack.go

```diff
@@ -0,0 +1,83 @@
+package framework
+
+import (
+	"fmt"
+	"github.com/konveyor/forklift-controller/tests/suit/utils"
+)
+
+func (r *OpenStackClient) SetupClient(vmName string, networkName string, volumeTypeName string) (err error) {
+	r.vmData.testVMName = vmName
+	r.vmData.testNetworkName = networkName
+	r.vmData.testVolumeName = volumeTypeName
+	return nil
+}
+
+// LoadSourceDetails - Load Source VM details from oVirt
+func (r *OpenStackClient) LoadSourceDetails(f *Framework, namespace string, contName string) (vm *OpenStackVM, err error) {
+
+	pod, err := utils.FindPodByPrefix(f.K8sClient, namespace, contName, fmt.Sprintf("app=%s", contName))
```

Comment:
it could be better to change it with the more common pattern of using a label selector but that's unrelated to this PR